datasets/snli/README.md

```diff
@@ -0,0 +1,140 @@
+
+The authors report 3/5 annotator agreement on 98% of the validation set and unanimous annotator agreement on 58.3% of the validation set. If a label was chosen by three annotators, that label was made the gold label. Following from this, 2% of the data did not have a consensus label and was labeled '-' by the authors. 
+
+Label | Fleiss κ
+------|---------
+_contradiction_ | 0.77
+_entailment_ | 0.72
+_neutral_ | 0.60
+overall | 0.70
+
+## Dataset Structure
+
+### Splits, features, and labels
+
+The SNLI dataset has 3 splits: _train_, _validation_, and _test_. 
+Dataset Split | Number of Instances in Split
+--------------|--------------------------------------------
+Train | 550,152
+Validation | 10,000
+Test | 10,000
+
+Each data instance contains the following features: _premise_, _hypothesis_, _label_.
+
+Feature | Mean Token Count
+--------|-----------------
+Premise | 14.1
+Hypothesis | 8.3
+
+The _label_ has 4 possible values, _0_, _1_, _2_, _-_.  which correspond to _entailment_, _neutral_, _contradiction_, and _no label_ respectively. The dataset was developed so that the first three values would be evenly distributed across the splits. See the Annotation Process section for details on _no label_.
+
+### Span indices
+
+No span indices are included in this dataset.
+
+### Example ID
+
+The ID is an integer starting from 0. It has no inherent meaning. 
+
+### Free text description for context (e.g. describe difference between title / selftext / body in Reddit data) and example
+
+For each ID, there is a string for the premise, a string for the hypothesis, and an integer for the label. Note that each premise appears three times with a different hypothesis and label. See the [SNLI corpus viewer](https://huggingface.co/nlp/viewer/?dataset=snli) to explore more examples.
+
+ID | Premise | Hypothesis | Label
+---|---------|------------|-------
+0	| Two women are embracing while holding to go packages.	| The sisters are hugging goodbye while holding to go packages after just eating lunch.	| 1
+1	| Two women are embracing while holding to go packages.	| Two woman are holding packages.	| 0
+2	| Two women are embracing while holding to go packages.	| The men are fighting outside a deli. | 2
+
+### Suggested metrics / models:
+
+[SemBERT](https://arxiv.org/pdf/1909.02209.pdf) (Zhousheng Zhang et al, 2019b) is currently listed as SOTA, achieving 91.9% accuracy on the test set. See the [corpus webpage](https://nlp.stanford.edu/projects/snli/) for a list of published results.
+
+## Known Limitations
+### Known social biases
+
+The language reflects the content of the photos collected from Flickr, as described in the Data Collection section.
```

Comment:
There's a paper that quantifies some kinds of bias: https://www.aclweb.org/anthology/W17-1609/